packages/aws-cdk/lib/api/aws-auth/awscli-compatible.ts

```diff
@@ -41,11 +42,11 @@ export class AwsCliCompatible {
     ];
 
     if (await fs.pathExists(credentialsFileName())) {
-      sources.push(() => new AWS.SharedIniFileCredentials({ profile, filename: credentialsFileName() }));
+      sources.push(() => new AWS.SharedIniFileCredentials({ profile, filename: credentialsFileName(), tokenCodeFn }));
     }
 
     if (await fs.pathExists(configFileName())) {
-      sources.push(() => new AWS.SharedIniFileCredentials({ profile, filename: credentialsFileName() }));
+      sources.push(() => new AWS.SharedIniFileCredentials({ profile, filename: credentialsFileName(), tokenCodeFn }));
```

Comment:
Oh, this block slipped in accidentally during a refactor of mine. 

Mind taking it out? <3

Reply:
Sure thing :) 

That line looked weird to me, but as it did not seem to cause issue when I was testing it I decided to keep it.